frontend/packages/client/src/components/ProposalCreate/StepTwo.js

```diff
@@ -47,7 +47,10 @@ const StepTwo = ({ stepData, setStepValid, onDataChange }) => {
 
     // push now if date is today and not already in time interval
     if (cutOffDate) {
-      const nowDate = new Date();
+      const nowDate =
+        process.env.APP_ENV === 'PRODUCTION'
+          ? new Date(Date.now() + 60 * 60 * 1000) // delay by an hour in prod env
+          : new Date();
       nowDate.setSeconds(0);
```

Comment:
I think this only applies for the ones that will make use of the snapshot process. 
If strategy for the proposal is `one-address-one-vote does` does it make sense to start the voting one hour later? what do you think @dbslone? This would require to see what strategy was selected on the previous step and checking the environment.